StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentOption+Images.swift

```diff
@@ -97,6 +97,24 @@ extension STPPaymentMethod {
             return makeIcon()
         }
     }
+
+    /// Returns an image to display inside a row representing the given payment option in the saved PM row view
+    func makeSavedPaymentMethodRowImage() -> UIImage {
```

Comment:
`makeSavedPaymentMethodCellImage` uses carousel images for card brands and for SEPA. We need the images that are used in the PAN field for card brand rather than the carousel images. Here the diff if we use `makeSavedPaymentMethodCellImage`, notice visa & SEPA look a little different than what is laid out in the mocks. The differences would be more sark in dark mode but I don't think we have mocks for dark mode.
![CleanShot 2024-05-09 at 17 57 13](https://github.com/stripe/stripe-ios/assets/88012362/b2e25ddf-151a-43ac-8792-d4314933b16d)


#### Mock
![CleanShot 2024-05-09 at 17 58 11](https://github.com/stripe/stripe-ios/assets/88012362/22725410-0a47-47a4-871f-6254d0d47ce0)

Reply:
Ohh right, we have 'carousel' versions of these. I guess we can delete those once we finish LPM visibility and the horizontal saved pm carousel goes away.

Reply:
Ohh right, we have 'carousel' versions of these. I guess we can delete those once we finish LPM visibility and the horizontal saved pm carousel goes away.